Daily_Reward/DailyReward.txt

```diff
@@ -44,17 +45,23 @@ OnPCLoginEvent:
 	if(.IPCheck){
 		query_sql("SELECT last_ip FROM `login` WHERE account_id = "+getcharid(3)+"", .@LastIp$);
 		query_sql("SELECT account_id FROM `login` WHERE last_ip = '"+.@LastIp$+"'", .@AccountId);
-		if(getarraysize(.@AccountId) > .IPCheck){
+		for(.@i=0;.@i<getarraysize(.@AccountId);.@i++){
+			query_sql("SELECT `value` FROM `acc_reg_num` WHERE `account_id` = '"+.@AccountId[.@i]+"' AND `key` = '#LastDailyReward'",.@LastIp2);
+			if(#LastDailyReward < .@LastIp2){
 			message strcharinfo(0),"[Daily Rewards]: Rewards are limited to "+.IPCheck+" per IP sorry.";
```

Comment:
should be
`message strcharinfo(0),"[Daily Rewards]: Rewards are limited to 1 per IP sorry.";`

since you no longer compare size of ip check